docs/tutorials/entrypoints.rst

```diff
@@ -8,10 +8,96 @@
 Entry points
 ************
 
-
 .. todo::
 
-   - Mention what is the purpose of entry points
-   - Give an example of a console entry point
    - Give an example of a custom entry point
      - Mention pluggy maybe for an example use-case?
+
+Introduction
+============
+
+Entry points provide a mechanism to advertise components of an installed
+distribution to other code or users. Most notably, the ``console_scripts``
+entry points will create a command line wrapper.
+
+See the `PyPA documentation on entry points <https://packaging.python.org/en/latest/specifications/entry-points/>`_.
```

Comment:
Those docs are actually outdated and confusing, so only linking to the `entry-points` section of https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#declaring-project-metadata is better I'd say.

Reply:
This part of the documentation is way better compared to the entry point specification.

I think the entry point specification is not outdated but just confusing. Apparently the entry point group is still called `console_scripts` but the table is called `scripts`.